docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -54,12 +54,20 @@ You can find the solution in the `security-jpa-quickstart` link:{quickstarts-tre
 
 == Create and verify the Maven project
 
+ifndef::quarkus-security-jpa-reactive[]
 For Quarkus Security to be able to map your security source to Jakarta Persistence entities, ensure that the Maven project in this tutorial includes the `security-jpa` or `security-jpa-reactive` extension.
+endif::quarkus-security-jpa-reactive[]
+ifdef::quarkus-security-jpa-reactive[]
+For Quarkus Security to be able to map your security source to Jakarta Persistence entities, ensure that the Maven project in this tutorial includes the `security-jpa` extension.
```

Comment:
```suggestion
For Quarkus Security to be able to map your security source to Jakarta Persistence entities, ensure that the Maven project in this tutorial includes the `security-jpa` or `security-jpa-reactive` extension.
```

Reply:
I think this feedback might be incorrect. 
The ifdef statement is visible only in the product documentation. (We define the attribute only in the product docs.)
In that case, for the ifdef statement, we want the preview (unsupported) `security-jpa-reactive` extension to be absent.